src/Evolution/Systems/CurvedScalarWave/Equations.cpp

```diff
@@ -40,12 +40,15 @@ db::const_item_type<FieldTag> weight_char_field(
 }
 
 template <size_t Dim>
-db::const_item_type<Tags::CharacteristicFields<Dim>> weight_char_fields(
-    const db::const_item_type<Tags::CharacteristicFields<Dim>>& char_fields_int,
-    const db::const_item_type<Tags::CharacteristicSpeeds<Dim>>& char_speeds_int,
-    const db::const_item_type<Tags::CharacteristicFields<Dim>>& char_fields_ext,
-    const db::const_item_type<Tags::CharacteristicSpeeds<Dim>>&
-        char_speeds_ext) noexcept {
+using char_field_tags =
+    tmpl::list<Tags::VPsi, Tags::VZero<Dim>, Tags::VPlus, Tags::VMinus>;
```

Comment:
This list should be in a header file, perhaps `Tags.hpp` that also has `Tags::CharacteristicFields`, because the list defines the ordering of these tags in the `Variables`. E.g. `Characteristics.hpp` depends on that ordering.